contributingGuides/BUGZERO_CHECKLIST.md

```diff
@@ -39,7 +40,7 @@ Who reported the bug:
 
 <details>
 <summary>Regression Test Proposal Template</summary>
-<!-- AFTER FILLING THIS OUT, be sure to remove the <details> tags!!!!! -->
+<!-- AFTER FILLING THIS OUT, be sure to remove the <details> and <summary> tags from this part of the checklist!!!!! -->
```

Comment:
I added this because I saw sometimes people were leaving the summary tags which messes up the rendering.